src/EntityFramework.SqlServerCompact/EntityFramework.SqlServerCompact.nuspec

```diff
@@ -8,6 +8,10 @@
         <dependency id="EntityFramework" version="$Version$" exclude="Build,Analyzers" />
         <dependency id="Microsoft.SqlServer.Compact" version="$MicrosoftSqlServerCompactVersion$" exclude="Build,Analyzers" />
       </group>
+      <group targetFramework=".NETFramework4.5">
```

Comment:
This caused a nuget warning when I updated the SDK. Was this just an oversight? @ajcvickers @divega ?

Reply:
@JunTaoLuo what is the warning?

@bricelam FYI.

Reply:
Oversight. Fixed it in the main package. Missed it here.